CITATIONS.md

```diff
@@ -10,23 +10,26 @@
 
 ## Pipeline tools
 
-* [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
+- [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
 
```

Comment:
I think you should add here the other tools you are using the pipeline ? `bowtie2`, `macs2`, `seacr`, etC.

Reply:
Good point, I will add these in